tools/pydfu.py

```diff
@@ -601,7 +622,18 @@ def main():
         command_run = True
 
     if args.path:
-        elements = read_dfu_file(args.path)
+        if str(args.path).endswith(".bin"):
+            if not args.address:
+                return
```

Comment:
It would possibly be helpful to have an error message here if the address hasn't been supplied?

Reply:
Yes, it would

Reply:
solved